ansible/roles/cleaner/tasks/main.yml

```diff
@@ -160,29 +63,3 @@
   include_tasks: scenarios/srm-toolchain.yml
   when: install_srm_toolchain == true
   tags: srm_toolchain
-
-- name: clean all configuration and log files
```

Comment:
Is this being taken care of elsewhere?
These two blocks clean the directories created.

Reply:
Delfin cleanup in delfin ansible play